app/views/admin/edition_world_tags/edit.html.erb

```diff
@@ -1,12 +1,13 @@
-<% page_title "Edit topics: " + @edition.title %>
+<% content_for :page_title, "#{@edition.title}: Worldwide tags" %>
+<% content_for :context, @edition.title %>
+<% content_for :title, "Worldwide tags" %>
 
-<div class="row">
-  <h1><%= @edition.title %></h1>
-</div>
-<div class="row">
-  <div class="col-md-12">
-    <h2>Worldwide</h2>
-    <hr>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-full">
+    <%= render "govuk_publishing_components/components/heading", {
+      text: "Selected topics",
```

Comment:
Should we name this, `Selected world tags` rather than `Topics`?

Reply:
Potentially? "Selected topics" is like for like with the bootstrap page so I've just used that